pkgs/top-level/python-packages.nix

```
@@ -3674,6 +3674,8 @@ in {
 
   moderngl = callPackage ../development/python-modules/moderngl { };
 
+  moderngl_window = callPackage ../development/python-modules/moderngl_window { };
```

```suggestion
  moderngl-window = callPackage ../development/python-modules/moderngl_window { };
```

As per https://www.python.org/dev/peps/pep-0503/#normalized-names

---

Ok, good to know! Does this also mean that I should update the directory name?